pkg/security/controller/namespace_scc_allocation_controller.go

```diff
@@ -226,6 +235,21 @@ func allocateNextContiguousBit(allocated *big.Int, max int) (int, bool) {
 	return 0, false
 }
 
+func (c *NamespaceSCCAllocationController) AddNextPeriodicRepair() {
+	// For this controller to work correctly we need to ensure a periodic repair
+	// of all the range allocations. For that we are adding an artificial key
+	// which will trigger that every 8 hours.
+
+	// ****************************************************************************
+	// For the development cycle, make the periodic repair faster (every 5 minutes)
+	delta := time.Duration(5 * time.Minute)
```

Comment:
Were we not going to add some sort of global private config that overrides this?

Reply:
That was my initial idea, but I dropped it. This mechanism proved to work just fine in kube-controller-manager-operator.